iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/OperationalLimitsHolderImpl.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.*;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+class OperationalLimitsHolderImpl implements OperationalLimitsOwner {
+
+    private final EnumMap<LimitType, OperationalLimits> operationalLimits = new EnumMap<>(LimitType.class);
+    private final AbstractIdentifiable<?> identifiable;
+    private final String attributeName;
+
+    OperationalLimitsHolderImpl(AbstractIdentifiable<?> identifiable, String attributeName) {
+        this.identifiable = Objects.requireNonNull(identifiable);
+        this.attributeName = Objects.requireNonNull(attributeName);
+    }
+
+    @Override
+    public void setOperationalLimits(LimitType limitType, OperationalLimits operationalLimits) {
+        OperationalLimits oldValue = this.operationalLimits.put(limitType, operationalLimits);
+        identifiable.getNetwork().getListeners().notifyUpdate(identifiable, attributeName + "_" + limitType, oldValue, operationalLimits);
+    }
+
+    List<OperationalLimits> getOperationalLimits() {
+        return Collections.unmodifiableList(new ArrayList<>(operationalLimits.values()));
+    }
```

Comment:
Do you really need to return a List, or a Collection could be OK?

Reply:
A collection would be okay, I will modify this

Reply:
Just to be sure: do you mean in the API or in the implementation?